src/main/java/org/bookstore/model/CartItem.java

```diff
@@ -0,0 +1,30 @@
+package org.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "cart_items")
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @JoinColumn(name = "shopping_cart_id", nullable = false)
+    @ManyToOne(fetch = FetchType.LAZY)
+    private ShoppingCart shoppingCart;
+    @JoinColumn(name = "book_id", nullable = false)
+    @OneToOne(fetch = FetchType.LAZY)
+    private Book book;
```

Comment:
pls add `optional = false` since cart item can't exits without book

also make sure that when you delete a book - corresponding order items are deleted as well